package.json

```diff
@@ -54,8 +51,11 @@
     "fast-lorem-ipsum": "1.2.0",
     "husky": "9.0.11",
     "jest": "29.7.0",
+    "jest-junit": "16.0.0",
     "lint-staged": "15.2.2",
     "prettier": "3.2.5",
+    "ts-jest-resolver": "2.0.1",
+    "tsx": "4.7.1",
```

Comment:
why do we need `tsx` here?

Reply:
<img width="267" alt="image" src="https://github.com/shelfio/fast-uslug/assets/31747499/e15b14ee-92bc-4223-895e-a0c5317a3780">